### PR TITLE
Support econia v3

### PR DESCRIPTION
**Pull request by Manahip**

### CODE REVIEW

File: sources/aggregator.move [hunk 1]
```diff
@@ -46,7 +45,7 @@ module hippo_aggregator::aggregator {
         time_stamp: u64
     }
 
-    public entry fun initialize(admin: &signer) {
+    fun init_module(admin: &signer) {
```
Comment (Manahip) [inline sources/aggregator.move:48]: I think this needs to be an entry function?

File: sources/devnet.move [hunk 2]
```diff
@@ -25,17 +24,19 @@ module hippo_aggregator::devnet {
     }
 
     #[cmd(desc=b"Create BTC-USDC pool on econia and add liquidity")]
-    public entry fun mock_deploy_econia(admin: signer) {
+    public entry fun mock_deploy_econia(admin: signer, market_id: u64) {
         use econia::market;
         use econia::user;
-        market::register_market<BTC, USDC, E0>(&admin);
-        user::register_market_account<BTC, USDC, E0>(&admin, 0);
+        let lot_size: u64 = 1000;
```
Comment (Manahip) [inline sources/devnet.move:30]: can we use lot_size = 1, tick_size = 1000?

Comment (Manahip) [reply sources/devnet.move:30]: scratch that. that doesn't make sense. Try lot_size = 1000, tick_size = 10000

File: sources/aggregator.move (hunk 1 above)
Comment (robbinh) [reply sources/aggregator.move]: Have changed it to `entry` function

File: sources/devnet.move (hunk 2 above)
Comment (robbinh) [reply sources/devnet.move:30]: As discussed, use 1000 for both

**Pull request merged**